test/soft-redirects.js

```diff
@@ -67,7 +67,9 @@ describe('Soft Redirects', function() {
 
 });
 
-describe('Soft Redirects - with `statusCode` option', async() => {
+describe('Soft Redirects - with `statusCode` option', async function() {
```

Comment:
Arrow functions break the way mocha uses "this" which prevents calls to "this.timeout".

Reply:
It fails silently, too. Fun.